lib/cmd/fhc/target.js

```diff
@@ -58,73 +64,81 @@ function target (argv, cb) {
     var where = ini.get("global") ? "global" : "user";
     ini.set('feedhenry', tar, where);
     ini.save(function(err) {
-      if (err) return cb(err);
+      if (err) {
+        return cb(err);
+      }
 
       if (args.length === 3) {
         return login({ _ : [args[1], args[2]] }, cb);
-      }else {
-        // attempt to set existing target cookie..
-        var t = {
-          target: tar
-        };
-        if(args[1]) t['user'] = args[1];
+      }
```

Comment:
Remove unnecessary `else` due to short-circuit. Indentation change makes the diff weird.